docs/_dapp/native-accounts.md

```diff
@@ -3,140 +3,107 @@ title: Go Account Management
-should be interested in doing is account management.
-
-Although all current leading Ethereum implementations provide account management built in,
-it is ill advised to keep accounts in any location that is shared between multiple
-applications and/or multiple people. The same way you do not entrust your ISP (who is
-after all your gateway into the internet) with your login credentials; you should not
-entrust an Ethereum node (who is your gateway into the Ethereum network) with your
-credentials either.
-
-The proper way to handle user accounts in your native applications is to do client side
-account management, everything self-contained within your own application. This way you
-can ensure as fine grained (or as coarse) access permissions to the sensitive data as
-deemed necessary, without relying on any third party application's functionality and/or
-vulnerabilities.
-
-To support this, `go-ethereum` provides a simple, yet thorough accounts package that gives
-you all the tools to do properly secured account management via encrypted keystores and
-passphrase protected accounts. You can leverage all the security of the `go-ethereum`
-crypto implementation while at the same time running everything in your own application.
+Geth provides a simple, yet thorough accounts package that includes all the tools developers
+need to leverage all the security of Geth's crypto implementation in a Go native application.
+The account management is done client side with all sensitive data held inside the application.
+This gives the user control over access permissions without relying on any third party.
 
-## Encrypted keystores
+{:toc}
 
-Although handling accounts locally to an application does provide certain security
-guarantees, access keys to Ethereum accounts should never lay around in clear-text form.
-As such, we provide an encrypted keystore that provides the proper security guarantees for
-you without requiring a thorough understanding from your part of the associated
-cryptographic primitives.
+-   this will be removed by the toc
 
-The important thing to know when using the encrypted keystore is that the cryptographic
-primitives used within can operate either in *standard* or *light* mode. The former
-provides a higher level of security at the cost of increased computational burden and
-resource consumption:
+## Encrypted keystores
 
- * *standard* needs 256MB memory and 1 second processing on a modern CPU to access a key
- * *light* needs 4MB memory and 100 millisecond processing on a modern CPU to access a key
+Access keys to Ethereum accounts should never be stored in plain-text. Instead, they should be 
+stored encrypted so that even if the mobile device is accessed by a malicious third party the 
+keys are still hidden under an additional layer of security. Geth provides a keystore that enables 
+developers to store keys securely using the [`secp256k1`][secp256k1] elliptic curve, 
```

Comment:
The keystore uses Scrypt to store the secp256k1 keys. So this phrasing is a bit off.